nslsii/iocs/tests/temperature_controllers_test.py

```diff
@@ -26,14 +27,22 @@ def test_Eurotherm(RE):
     stdin = None
 
     # Start up an IOC based on the thermo_sim device in caproto.ioc_examples
+    '''
     ioc_process = subprocess.Popen([sys.executable, '-m',
                                     'caproto.tests.example_runner',
                                     'caproto.ioc_examples.thermo_sim'],
                                    stdout=stdout, stdin=stdin,
                                    env=os.environ)
+    '''
+    ioc_process = subprocess.Popen([sys.executable, '-m',
```

Comment:
Can you provide an explanation for why this change is necessary and remove the commented out section above?

Reply:
it removed an additional indirection. Today, temperature_contollers_test however failed in both cases (with and without caproto.tests.example_runner) even when run as the first test. Therefore, in this PR it was disabled by renaming the corresponding file.

Reply:
OK so I don't follow this, and what I take form this line:
> Therefore, in this PR it was disabled by renaming the corresponding file.

is that we are now no longer testing against that test which we will need to resolve before merging.

I am not comfortable with renaming the test and placing it in `nslsii/iocs/tests` as this is not a test of the ioc but a test of the ophyd class, which logically belongs in `nslsii/tests/`.

Does it help if we move all of the tests (even the ioc tests) to `nslsii/tests`? It is our standard way of arranging the repos, with all tests in `nslsii/tests` and then seperated into modules (say a single file for all `ioc tests`).

Reply:
all files have been moved to nslsii/tests and successfully tested